### ISSUE TITLE
Allow xdebug to accept remote connections

### ISSUE THREAD

**Issue by edurenye** [wadmiraal/docker-drupal#34]
Right now the debugging does not work, so we need to fix that.

**Comment by wadmiraal** [comment]
Can you elaborate? I admit I never really used XDebug, so I wouldn't know where to start.

**Comment by edurenye** [comment]
Yes, I think the xdebug config just needs to set enable remote connections option to TRUE, but I found https://docs.devwithlando.io/ and it's more customizable, xdebug works there and is really easy to use and I will switch to that. You could give it a look, maybe you could also find it useful for you and do not need to maintain a separate tool.
